ui/scss/user-dashboard/_admin.scss

```diff
@@ -1114,4 +1114,8 @@ table {
 
 }
 
-
+.username.trim {
```

Comment:
Have a look if there’s a common sass file for all the spaces that need this to locate this. create it otherwise